# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/nethogs/hotfix-ver.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff -urN nethogs-0.9.0.orig/determineVersion.sh nethogs-0.9.0/determineVersion.sh
--- nethogs-0.9.0.orig/determineVersion.sh	2026-07-26 06:58:35.766358292 +0200
+++ nethogs-0.9.0/determineVersion.sh	2026-07-26 06:58:35.771234428 +0200
@@ -1,6 +1,7 @@
 #!/bin/bash
-if [ $(git rev-parse --show-toplevel 2>/dev/null) == $(pwd) ]; then
+gitroot=$(git rev-parse --show-toplevel 2>/dev/null)
+if [ -n "$gitroot" ] && [ "$gitroot" = "$(pwd)" ]; then
   git describe --tags 2>/dev/null | tail -c +2
 else
-  pwd | sed -e "s/.*\\///" | sed -e "s/[^-]*//" | tail -c +2
+  basename "$(pwd)" | sed 's/^[^-]*-//'
 fi
